handbook/engineering/web/extensibility/index.md

```diff
@@ -11,41 +11,48 @@ The extensibility team owns our code host and third-party integrations (includin
   - [TJ Kandala](../../../../company/team/index.md#tharuntej-kandala-he-him)
   - [Murat Sutunc](../../../../company/team/index.md#murat-sutunc-he-him)  
 
-## Contact
+## Mission
 
-- [#extensibility-chat](https://app.slack.com/client/T02FSM7DL/C01LZKLRF0C) channel or @extensibility on Slack
-- On GitHub, mention or assign issues to the [@sourcegraph/extensibility](https://github.com/orgs/sourcegraph/teams/extensibility) team.
+Our mission is to bring the value of Sourcegraph to everywhere you work with code and to bring the value of other developer tools into Sourcegraph.
 
-## Goals
+## Vision
 
-TODO - link to Productboard roadmap
+Make Sourcegraph part of developers' daily workflow.
 
-## Ownership
+1. We'll build a healthy ecosystem of Sourcegraph extensions to bring the power of existing developer tools into our platform to empower developers.
+1. We'll augment existing developer workflows by building new ways to use Sourcegraph on other websites and IDEs.
 
-This team is responsible for:
+## Responsibilities
 
```

Comment:
```suggestion
_The extensibility team has many ownership areas, but not all of them are under active development at the same time. We nonetheless list them all here as a source of truth for deciding if an issue, feedback, or comment is relevant to the extensibility team. You can find our current priorities in our [goals page](goals.md)._
```
Doesn't have to be this wording, but would like to call out that we own lots of things, but don't assume we're actively working on these things. It's still useful to describe them all because if a bug or feedback arises that's relevant to any of them, we want people to know they should send to us!